visivo/commands/serve_phase.py

```diff
@@ -85,7 +86,7 @@ def serve_phase(output_dir, working_dir, default_target, name_filter, threads):
         threads=threads,
     )
 
-    def cli_changed():
+    def cli_changed():  # TODO: Include changes to cmd models
```

Comment:
@tgsoverly I think we should watch for changes to models in addition to traces and update downstream traces when those changes occur. I think people will expect that behavior based on how traces update when changed. This type of behavior is particularly useful for user's that do a lot of calculation in the models (ie rwx).